mariadb/templates/etc/conf.d/server.cnf.j2

```diff
@@ -0,0 +1,99 @@
+# {{ ansible_managed }}
+#
+# These groups are read by MariaDB server.
+# Use it for options that only the server (but not clients) should see
+
+[mysqld]
+#
+# * Basic Settings
+#
+user                    = mysql
+pid-file                = {{ mariadb_run_dir }}/mysqld.pid
+socket                  = {{ mariadb_socket }}
+port                    = {{ mariadb_bind_port }}
+basedir                 = /usr
+datadir                 = {{ mariadb_datadir }}
+tmpdir                  = /tmp
+skip-external-locking
+
+# Instead of skip-networking the default is now to listen only on
+# localhost which is more compatible and is not less secure.
+bind-address            = {{ mariadb_bind_address }}
+
+#
+# * Fine Tuning
+#
+key_buffer              = 16M
+max_allowed_packet      = 16M
+thread_stack            = 192K
+thread_cache_size       = 8
+# This replaces the startup script and checks MyISAM tables if needed
+# the first time they are touched
```

Comment:
Can you align all values (equal symbols)?

Reply:
All values are aligned, if a `#` is at the beginning of the line, then the `=` is one column right of a line without `#`. So if you remove the `#` all `=` are aligned.